LivingCity/traffic/b18CommandLineVersion.cpp

```diff
@@ -169,6 +180,122 @@ void B18CommandLineVersion::runB18Simulation() {
+          std::cout << "Successfully opened file: " << busLinesPath << std::endl;
+      }
+      bool isFirstLine = true;
+      while (std::getline(file, line)) {
+
+        if (isFirstLine) {
+            isFirstLine = false;
+            continue; // Skip the header line
+        }
+
+          std::stringstream ss(line);
+          std::string value;
+          std::vector<int> stops;
+
+          std::getline(ss, value, ','); // Skip trip_id
+          std::getline(ss, value, ','); // Skip block_id
+          std::getline(ss, value, ','); // Skip direction_id
+          std::getline(ss, value, ','); // Skip shape_id
+
+          std::getline(ss, value, ','); // Read route_id
+          try {
+              busRouteIds.push_back(std::stoi(value));
+          } catch (const std::invalid_argument& e) {
+              std::cerr << "Invalid route_id: " << value << " in line: " << line << std::endl;
+              continue;
+          }
+
+          std::getline(ss, value, ','); // Skip route_short_name
+
+          std::getline(ss, value, ','); // Read departure_time
+          try {
+              busDepartureTimes.push_back(std::stoi(value));
+          } catch (const std::invalid_argument& e) {
+              std::cerr << "Invalid departure_time: " << value << " in line: " << line << std::endl;
+              continue;
+          }
+
+          std::getline(ss, value); // Read bus_stop_osmid
+
+          // Extract the content within double quotes and remove square brackets
+          size_t start = value.find('"');
+          size_t end = value.find_last_of('"');
+          if (start != std::string::npos && end != std::string::npos && end > start) {
+              std::string stops_str = value.substr(start + 1, end - start - 1);
+              //std::cout << "Extracted stops_str: " << stops_str << std::endl; // Debug: Print extracted stops_str
+
+              // Remove square brackets
+              stops_str.erase(std::remove(stops_str.begin(), stops_str.end(), '['), stops_str.end());
+              stops_str.erase(std::remove(stops_str.begin(), stops_str.end(), ']'), stops_str.end());
+
+              std::stringstream stopStream(stops_str);
+              std::string stop;
+
+              while (std::getline(stopStream, stop, ',')) {
+                  // Remove leading and trailing spaces
+                  stop.erase(0, stop.find_first_not_of(' '));
+                  stop.erase(stop.find_last_not_of(' ') + 1);
+                  //std::cout << "Parsed stop: " << stop << std::endl; // Debug: Print each parsed stop
+                  try {
+                      stops.push_back(std::stoll(stop));
+                  } catch (const std::invalid_argument& e) {
+                      std::cerr << "Invalid stop_id: " << stop << " in line: " << line << std::endl;
+                  }
+              }
+          } else {
+              std::cerr << "Invalid bus_stop_osmid: " << value << " in line: " << line << std::endl;
+          }
```

Comment:
@SLZ0106
Luze, if you dont mind, check if the code works here sicne it failed to detect my stations

Successfully opened file: bus_schedules.csv
Invalid stop_id: "154782 in line: 8617345_merged_8859463 ,a_101    ,0,167662,14287,1                ,15840,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65295383 in line: 8617345_merged_8859463 ,a_101    ,0,167662,14287,1                ,15840,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "154782 in line: 8617345_merged_8859463 ,a_101    ,0,167662,14287,1                ,15840,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid stop_id: "401485 in line: 8617552_merged_8848393 ,a_101    ,1,167664,14287,1                ,16920,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid routing_id: [258910949 in line: 8617552_merged_8848393 ,a_101    ,1,167664,14287,1                ,16920,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid routing_id: "401485 in line: 8617552_merged_8848393 ,a_101    ,1,167664,14287,1                ,16920,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid stop_id: "154782 in line: 8617308_merged_8872951 ,a_102    ,0,167662,14287,1                ,17580,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65295383 in line: 8617308_merged_8872951 ,a_102    ,0,167662,14287,1                ,17580,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "154782 in line: 8617308_merged_8872951 ,a_102    ,0,167662,14287,1                ,17580,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid stop_id: "154782 in line: 8617307_merged_8872954 ,a_103    ,0,167662,14287,1                ,18660,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65295383 in line: 8617307_merged_8872954 ,a_103    ,0,167662,14287,1                ,18660,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "154782 in line: 8617307_merged_8872954 ,a_103    ,0,167662,14287,1                ,18660,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid stop_id: "401485 in line: 8617551_merged_8848394 ,a_102    ,1,167664,14287,1                ,18660,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid routing_id: [258910949 in line: 8617551_merged_8848394 ,a_102    ,1,167664,14287,1                ,18660,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid routing_id: "401485 in line: 8617551_merged_8848394 ,a_102    ,1,167664,14287,1                ,18660,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid stop_id: "154782 in line: 8617306_merged_8872953 ,a_104    ,0,167662,14287,1                ,19380,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65295383 in line: 8617306_merged_8872953 ,a_104    ,0,167662,14287,1                ,19380,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "154782 in line: 8617306_merged_8872953 ,a_104    ,0,167662,14287,1                ,19380,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid stop_id: "164708 in line: 8617496_merged_8862511 ,a_101    ,0,167661,14287,1                ,19500,"[65324593, 65307352, 65307356, 65307359, 65305819, 65307363, 65307366, 65307376, 65307381, 65306087, 65307399, 65307404, 65307408, 65307412, 65307414, 258758544, 65307419, 173176455, 5283675522, 173176449, 65292743, 65307438, 65307440, 65295364, 65295367, 4019294259, 65295377, 65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","164708,543511,224631,159140,159142,159144,159146,158471,159149,159151,159153,159156,547161,159158,159160,158552,159163,159168,159171,159174,159176,159180,119754,159181,159183,37728,400639,37720,153531,159187,159191,159194,154758,154764,154768,35835,154772,154776,154780,154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65324593 in line: 8617496_merged_8862511 ,a_101    ,0,167661,14287,1                ,19500,"[65324593, 65307352, 65307356, 65307359, 65305819, 65307363, 65307366, 65307376, 65307381, 65306087, 65307399, 65307404, 65307408, 65307412, 65307414, 258758544, 65307419, 173176455, 5283675522, 173176449, 65292743, 65307438, 65307440, 65295364, 65295367, 4019294259, 65295377, 65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","164708,543511,224631,159140,159142,159144,159146,158471,159149,159151,159153,159156,547161,159158,159160,158552,159163,159168,159171,159174,159176,159180,119754,159181,159183,37728,400639,37720,153531,159187,159191,159194,154758,154764,154768,35835,154772,154776,154780,154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "164708 in line: 8617496_merged_8862511 ,a_101    ,0,167661,14287,1                ,19500,"[65324593, 65307352, 65307356, 65307359, 65305819, 65307363, 65307366, 65307376, 65307381, 65306087, 65307399, 65307404, 65307408, 65307412, 65307414, 258758544, 65307419, 173176455, 5283675522, 173176449, 65292743, 65307438, 65307440, 65295364, 65295367, 4019294259, 65295377, 65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","164708,543511,224631,159140,159142,159144,159146,158471,159149,159151,159153,159156,547161,159158,159160,158552,159163,159168,159171,159174,159176,159180,119754,159181,159183,37728,400639,37720,153531,159187,159191,159194,154758,154764,154768,35835,154772,154776,154780,154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid stop_id: "401485 in line: 8617550_merged_8848395 ,a_103    ,1,167664,14287,1                ,19740,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid routing_id: [258910949 in line: 8617550_merged_8848395 ,a_103    ,1,167664,14287,1                ,19740,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid routing_id: "401485 in line: 8617550_merged_8848395 ,a_103    ,1,167664,14287,1                ,19740,"[258910949, 65333130, 65307165, 65295499, 65295496, 65295489, 65295485, 65295480, 65295471, 65295458, 65295450, 65280072, 65295444, 65295438, 65295435, 65295434, 65295420, 65295411, 65295399, 65295388, 65295383, 65295377, 4019294259, 65295367, 65307443, 65307440, 65307438, 65292743, 173176449, 5283675522, 173176455, 65337003, 65337001, 65327171, 65336999, 65336993, 65336987, 65336976, 65336974, 65326726, 65332833, 65336961, 65336956, 65336953, 65305827, 65336947, 65327408, 65324593]","401485,167044,159027,159021,154925,154921,154917,154913,154909,153723,154904,154899,19777,154897,154891,154888,154885,154879,154875,21006,154873,154868,154863,154860,383155,154855,150594,154854,154848,154846,154841,154837,154831,151639,154829,151685,154823,154821,154816,154814,154809,154807,154804,154802,154799,154794,154789,154788,461880,154784,154781,154777,154773,35833,154769,154762,159200,159196,159192,159188,153532,37721,400640,37727,47040,168103,168100,119752,165466,168099,168096,168093,168089,168087,167571,165354,166954,547162,168084,168081,165947,168077,168072,158474,168070,167849,165544,168067"
Invalid stop_id: "154782 in line: 8617305_merged_8872956 ,a_105    ,0,167662,14287,1                ,19920,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65295383 in line: 8617305_merged_8872956 ,a_105    ,0,167662,14287,1                ,19920,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "154782 in line: 8617305_merged_8872956 ,a_105    ,0,167662,14287,1                ,19920,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid stop_id: "154782 in line: 8617304_merged_8872955 ,a_106    ,0,167662,14287,1                ,20460,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: [65295383 in line: 8617304_merged_8872955 ,a_106    ,0,167662,14287,1                ,20460,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
Invalid routing_id: "154782 in line: 8617304_merged_8872955 ,a_106    ,0,167662,14287,1                ,20460,"[65295383, 65295388, 65295399, 65295411, 65295420, 65295434, 65295435, 65295438, 65295444, 65280072, 65295452, 65295458, 65295471, 65295480, 65295485, 65295489, 65295496, 65295499, 65307165, 258910948, 258910949]","154782,461879,154786,154792,154795,154798,154801,154803,154806,154810,154813,154817,154818,154822,151684,154828,151638,154833,154836,154842,154844,154847,154853,150593,154858,383154,154859,154862,154867,154872,21005,154874,154878,154884,154887,154893,154896,19776,154898,154903,153724,154908,154912,154916,154920,154923,159023,159025,401484"
[TIME] Started: <Lane_Map_creation>.